<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanFormUi</class>
 <widget class="SlGfwDialogForm" name="ScanFormUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>372</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <widget class="PlotWidget" name="LeftPlotWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>48</y>
     <width>559</width>
     <height>126</height>
    </rect>
   </property>
  </widget>
  <widget class="PlotWidget" name="RightPlotWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>223</y>
     <width>559</width>
     <height>126</height>
    </rect>
   </property>
  </widget>
  <widget class="SlGfwLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>23</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Siemens AD CH Simplified</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string>background-color: rgb(255, 255, 0)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="text">
    <string>左铣削单元</string>
   </property>
  </widget>
  <widget class="SlGfwLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>198</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Siemens AD CH Simplified</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string>background-color: rgb(255, 255, 0)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="text">
    <string>右铣削单元</string>
   </property>
  </widget>
  <widget class="SlGfwLabel" name="label_43">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>353</y>
     <width>105</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Y电机实际位置</string>
   </property>
  </widget>
  <widget class="SlGfwLineEdit" name="R_MotorRealPosLineEdit">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>353</y>
     <width>51</width>
     <height>15</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="SlGfwLabel" name="label_44">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>353</y>
     <width>91</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Y标定位置</string>
   </property>
  </widget>
  <widget class="SlGfwLineEdit" name="RightCalibPosLineEdit">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>353</y>
     <width>51</width>
     <height>15</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="SlGfwLabel" name="label_45">
   <property name="geometry">
    <rect>
     <x>421</x>
     <y>353</y>
     <width>81</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Y当前位置</string>
   </property>
  </widget>
  <widget class="SlGfwLineEdit" name="RightCurPosLineEdit">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>353</y>
     <width>51</width>
     <height>15</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="SlGfwLineEdit" name="LeftCalibPosLineEdit">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>178</y>
     <width>51</width>
     <height>15</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="SlGfwLineEdit" name="LeftCurPosLineEdit">
   <property name="geometry">
    <rect>
     <x>508</x>
     <y>178</y>
     <width>51</width>
     <height>15</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="SlGfwLabel" name="label_46">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>179</y>
     <width>91</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Y标定位置</string>
   </property>
  </widget>
  <widget class="SlGfwLabel" name="label_47">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>180</y>
     <width>91</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Y当前位置</string>
   </property>
  </widget>
  <widget class="SlGfwLineEdit" name="L_MotorRealPosLineEdit">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>178</y>
     <width>51</width>
     <height>15</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="SlGfwLabel" name="label_48">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>179</y>
     <width>105</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Y电机实际位置</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SlGfwDialogForm</class>
   <extends>QWidget</extends>
   <header>slgfwdialogform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SlGfwLabel</class>
   <extends>QFrame</extends>
   <header>slgfwlabel.h</header>
  </customwidget>
  <customwidget>
   <class>SlGfwLineEdit</class>
   <extends>QFrame</extends>
   <header>slgfwlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">plotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>LeftCurPosLineEdit</tabstop>
  <tabstop>LeftCalibPosLineEdit</tabstop>
  <tabstop>L_MotorRealPosLineEdit</tabstop>
  <tabstop>RightCurPosLineEdit</tabstop>
  <tabstop>RightCalibPosLineEdit</tabstop>
  <tabstop>R_MotorRealPosLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
